core/eventbroker/src/lib/log/Logger.ts

```diff
@@ -0,0 +1,43 @@
+export class Logger {
```

Comment:
should we create a library that all core components include, instead of implementing the same class in different projects?

Reply:
Absolutely, but I would suggest to create a separate story for that